Cargo.toml

```diff
@@ -15,6 +15,11 @@ license = "Unlicense/MIT"
 [dependencies]
 docopt = "0.6.81"
 lazy_static = "0.2.1"
-prettytable-rs = "0.6.2"
+quickcheck = "0.3.1"
```

Comment:
Both `quickcheck` and `second_law` should be `dev-dependencies`, which means they'll only be required during tests.